content/rc/cloud-integrations/pulumi/_index.md

```diff
@@ -0,0 +1,99 @@
+---
+Title: The Pulumi Redis Cloud Resource Provider
+LinkTitle: Pulumi
+description: Explains how to use Pulumi to provision Redis Cloud resources
+weight: $weight
+alwaysopen: false
+categories: ["RC"]
+aliases: /rc/cloud-integrations/pulumi/
+         /rc/cloud-integrations/pulumi.md
+---
+
+## Introduction to Pulumi
+
+Pulumi has gained significant popularity within the developer community due to its innovative approach to infrastructure as code (IaC) and its ability to simplify and streamline the process of cloud resource provisioning and management. Unlike traditional IaC tools, Pulumi allows developers to write infrastructure code using familiar programming languages, such as Python, JavaScript, TypeScript, and Go, rather than using domain-specific languages.
+
+This developer-friendly approach makes it easier for software engineers to leverage their existing programming skills and apply them to infrastructure provisioning, making the learning curve smoother. Pulumi's support for multiple programming languages also provides flexibility and caters to a wider range of developers, enabling teams to collaborate effectively and work with their preferred language.
+
+## The Pulumi Redis Cloud Resource Provider
+
+With the recent availability of the [Redis Cloud Resource Provider](https://www.pulumi.com/registry/packages/rediscloud/), developers can now take advantage of this popular technology to create Redis Cloud resources in their favorite programming language. As of this writing, the Pulumi Redis Cloud Provider supports the following programming languages :
+
+* TypeScript
+* Python
+* C#
+* Java
+* Go
+* YAML
+
+The Pulumi Redis Cloud Provider is based upon the TerraForm Redis Cloud Provider, thus for those who have experience using the TerraForm provider should quickly grasp the concepts behind the Pulumi provider.
+
+{{<note>}}
+The provider supports the creation of "Flexible/Annual" subscriptions, but cannot be used to create "Fixed" subscriptions
+{{</note>}}
+
+## Pulumi Redis Cloud Provider Resources and Functions
+
+Pulumi `resources` represent the **fundamental units that make up cloud infrastructure**. A provider may make `functions` available in its SDK as well as resource types. These `functions` are often **used to acquire information that is not part of a resource**. 
```

Comment:
```suggestion
Pulumi resources represent the fundamental units that make up cloud infrastructure. A provider can make functions available in its SDK and resource types. These functions are often used to acquire information that is not part of a resource. 
```

Reply:
We don't usually format terminology differently unless it's a command used in the cli.